### PR TITLE
Set image attribute when creating new class object

### PR DESCRIPTION
**Pull request by MerdanDev**
I had problem with SQL error, that image attribute doesn't have default value when seeding my my table. For example:
$obj = new \App\Models\Product();
$obj->name = 'name';
$obj->image = 'image path';
$obj->save();

### CODE REVIEW

**Comment by tabacitu** [comment]
Haha very nifty @MerdanDev , I like it. 
Indeed, I see no reason why this shouldn't also support adding images by path. 

@pxpm / @promatik do you guys foresee any problems with this? I don't.

File: 4.1/crud-fields.md [hunk 1]
```diff
@@ -908,6 +908,10 @@ Class Product extends Model
             $public_destination_path = Str::replaceFirst('public/', '', $destination_path);
             $this->attributes[$attribute_name] = $public_destination_path.'/'.$filename;
         }
+	else
+        {
+            return $this->attributes['image'] = $value;
```
Comment (promatik) [inline 4.1/crud-fields.md:913]:
```suggestion
            return $this->attributes[$attribute_name] = $value;
```

File: 4.1/crud-fields.md [hunk 2]
```diff
@@ -908,6 +908,10 @@ Class Product extends Model
...
+        }
```
Comment (promatik) [inline 4.1/crud-fields.md:914]:
```suggestion
        else
        {
            return $this->attributes[$attribute_name] = $value;
        }
```

**Review by promatik — CHANGES_REQUESTED** [review]

**Comment by promatik** [comment]
I don't think this will bring any trouble for us, at least I've been using this on my `trait SaveMedia` for a long time 😅

image

File: 4.1/crud-fields.md (hunk 2 above)
Comment (promatik) [reply 4.1/crud-fields.md:914]: Hi @MerdanDev, thank you for this suggestion, I approve it 👌
I have just this small changes to apply, and I see you've deleted the repo from your side, so I cannot apply the changes 😅, can you please try to somehow recover it?

**Pull request merged**